yjit.rb

```diff
@@ -155,8 +155,10 @@ def self.dump_exit_locations(filename)
 
   # Return a hash for statistics generated for the `--yjit-stats` command line option.
   # Return `nil` when option is not passed or unavailable.
-  def self.runtime_stats()
-    Primitive.rb_yjit_get_stats
+  # If arg is a hash, populate stats into it.
```

Comment:
I get that you followed the `GC.stat` pattern, but do we really need this "If arg is a hash" interface now? As long as it populates all stats, it'd be so slow that removing a single allocation might not really matter. This seems like a relatively complicated API design, and it also seems to complicate the implementation.

Reply:
Great, thanks! As you noted I did this for consistency but I agree just taking a key would make it a lot simpler.